src/app/api/types.ts

```diff
@@ -0,0 +1,5 @@
+import { NextRequest, NextResponse } from "next/server";
+
+export type ResponseData<
```

Comment:
I added this generic response type so that we'd get database to UI type safety for all future API endpoints.